### ISSUE TITLE
ERROR CODE

### ISSUE THREAD

**Issue by NitroX28** [ESX-Org/esx_license#11]
hi dude i have a problem with your ressource , but i don't know from where the error code is this 👍 

SCRIPT ERROR: @esx_license/server/main.lua:72: attempt to index a nil value (field '?')
> ref (@esx_license/server/main.lua:72)
>  (@mysql-async/mysql-async.js:14959)

**Comment by HezimX** [comment]
yep have the same error

**Comment by JingJayYu** [comment]
you need to go to your license in your database add your license like
   type        label
   dmv    CarLicense

**Comment by PanamaCityRP** [comment]
me too, it's resolved ?

**Comment by yosefij159** [comment]
i think the error is because you added a license but then you removed it.
maybe a player has a license (in user_licenses table) that you don't have anymore (in licenses table)

**Comment by HezimX** [comment]
yes in forgot to replay here but i had removed the driver license thats why i had that error.

**Comment by PanamaCityRP** [comment]
Oh ok i test that after ! Thx guys

**Comment by cold-cat** [comment]
@NitroX28 Same problem

**Comment by prouargegamer** [comment]
did you fix it?
